lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -5,6 +5,9 @@ module Engine
     module Analyzers
       class EngineConfig
         DEFAULT_COUNT_THRESHOLD = 2
+        IDENTICAL_CODE_CHECK = "identical_code".freeze
```

Comment:
💄 these are dashes in the config, not underscores. https://github.com/codeclimate/rfc/pull/79